test/fixtures/minio-fixture/docker-compose.yml

```diff
@@ -11,3 +11,14 @@ services:
     ports:
       - "9000"
     command: ["server", "/minio/data"]
+  minio-fixture-other:
```

Comment:
Possibly a silly question, but why do we need two Minio fixtures?

Reply:
Because the new QA project and one of its dependency (in this case `repository-s3`) are not allowed to declare the usage of the same fixture. This is prevented by the test framework as using the same fixture could cause issues when running tests in parallel using Gradle.

Reply:
Makes sense, thanks.